File: C_Program/while8.c
//wap to accept a no. and print the table of that no. using while loop.

#include<stdio.h>
int main()
{
	int num,res;
	int i=1;
	printf("\n enter the no.: ");
	scanf("%d",&num);
	while(i<=10)
	{
		res=num*i;
		printf("\n %d * %d = %d",num,i,res);
		i=i+1;
	}
}

File: Java_Programs/While8.java
//wap to accept a no and print table of that no.
import java.util.*;
class While8
{
	public static void main(String []args)
	{
		int num,i=1;

		Scanner s=new Scanner(System.in);

		System.out.println("Enter a number: ");
		num = s.nextInt();

		while(i<=10)
		{
			int res = num * i;
			System.out.println(num+ " * " +i+ " = " +res);
			i++;
		}

	}
}